sdk/src/test/java/io/opentelemetry/sdk/trace/RandomIdsGeneratorTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.trace;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import io.opentelemetry.trace.SpanId;
+import io.opentelemetry.trace.TraceId;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class RandomIdsGeneratorTest {
+
+  @Test
+  public void defaults() {
+    RandomIdsGenerator generator = new RandomIdsGenerator();
+
+    // Can't assert values but can assert they're valid.
+    TraceId traceId = generator.generateTraceId();
+    assertThat(traceId).isNotEqualTo(TraceId.getInvalid());
+
+    SpanId spanId = generator.generateSpanId();
+    assertThat(spanId).isNotEqualTo(SpanId.getInvalid());
```

Comment:
Maybe worth generating 1K ids to check?

Reply:
Sure - for the record, even with 1K test only takes 41ms on my macbook pro